pkg/util/date.go

```diff
@@ -724,3 +724,14 @@ func ConvertToPST(t time.Time) (*time.Time, error) {
 
 	return &pt, nil
 }
+
+// GetLocDateFromState gets the time based on the given state's timezone
+func GetLocDateFromState(state string) (*time.Time, error) {
+	tzString := StandardTimeZoneForState(state)
```

Comment:
@Swechhya pls review there. do we already have some existing function?

Reply:
@bidur-phil We do not have a similar function.